packages/cli/README.md

```diff
@@ -0,0 +1,32 @@
+# @uma/cli
+
+This package contains the UMA CLI tool. This tool works, but is deprecated, so its use is discouraged. It can be useful
+for certain tasks, like viewing governance/admin proposals, but it is usually preferred to use tools.umaproject.org or
+vote.umaproject.org.
+
+## Installing the package
+
+```bash
+yarn global add @uma/cli
+```
+
+## Running the package
+
+```bash
+uma-cli --network mainnet_mnemonic
+```
+
+## Using your own node URL
+
+By default, this package uses a default infura account that often exceeds its daily quota. To plug in your own node
+node ULR, export the following env variable:
+
+```bash
+export CUSTOM_NODE_URL=your.node.url.io
+uma-cli --network mainnet_mnemonic
```

Comment:
```suggestion
export MNEMONIC=your mnemonic 12 key words
uma-cli --network mainnet_mnemonic
```

Reply:
useful I guess to also show how to set your mnemonic here?

Reply:
I will wait to commit this suggestion until we decide whether we prefer:
```bash
MNEMONIC="some mnemonic" uma-cli --network mainnet_mnemonic
```
or
```bash
MNEMONIC="some mnemonic"
uma-cli --network mainnet_mnemonic
```

Reply:
personally I prefer option 2. cleaner distinction that these are separate commands.